CryptoTechReminderSystem/Gateway/SlackGateway.cs

```diff
@@ -53,7 +53,10 @@ public IList<SlackDeveloper> RetrieveDevelopers()
             var result = GetUsersAsync().Result;
             var users = result["members"];
 
-            return users.Select(developer => new SlackDeveloper
+            return users.Where(user => user["profile"]["email"] != null && 
```

Comment:
If the `SlackGateway` checks if a user has an email, does the `GetLateDevelopers` use case need to as well?